userenvs/components/05-rhubi-core-perl.json

```diff
@@ -0,0 +1,9 @@
+    {
+      "name": "core-perl",
+      "type": "manual",
+      "manual_info": {
+        "commands": [
+          "curl --silent -L https://cpanmin.us | /usr/bin/perl - App::cpanminus"
+        ]
+      }
+    }
```

Comment:
this seems like a duplicate of `userenvs/components/rhubi-core-perl.json` below and the name seems weird for this location...or am I missing something?